lib/index.ts

```diff
@@ -872,3 +872,9 @@ export class Stagehand {
       });
   }
 }
+
```

Comment:
these are all the types we also want public-facing; there's a few more that are just for internal uses, like `inference`